### PR TITLE
The Great Tribal Rework (Ready for Testing)

### PR DESCRIPTION
**Pull request by Tokorizo**
## About The Pull Request
The tribe is a flawed faction that I have given hundreds of hours playing to, It one of the most unique playstyles in the game but it constantly suffers flaws and errors due to whoever I'm running the faction. Be it the in-game leadership or just whoever maintains it. It always has problems and these have stemmed since DR in its creation to be an ERP faction by Zyy.

The removal of the tribe will pave a new role for people to try out called Tribal. 

### Role changes

The original Wayfarer tribal faction is going to be completely removed, lore-wise the Legion ended up destroying them and enslaving some of their population, a few managing to escape and be survivors in the waste. Mechanics wise the roles will be disabled in place for a new freeform role that allows players to make their own tribes.

Similar to Wastelander or Outlaw, Tribals start as neutral that have no ties or connections to any factions, they have the ability to form their own tribes similar to gangs and create IC diplomatic interactions. Playable solo or in a group.

In order to make them different from Outlaws and Wasters, they are given restrictions which include the ability and struggle to use certain workbenches like ammo and workshops. However, they have the ability to craft their own role-exclusive items with mob drops, unlocking a lot of the old tribal exclusive items more for the general player base to use. These items range from

- Deathclaw spears
- Tribal exclusive armors like light cloaks or tribal power armor
- War paints for cosmetics
- and more...

### Tribals

Basic information 
- Tribals start with **currently** 4 loadouts,
- Tribals are true neutral meaning they can act in evil or passive manner towards others. They can aid other factions like NCr or Legion for their own IC reasons. (Like a waster or outlaw)
### Pros
- Have a tribal language that lets them speak to fellow tribals in the wasteland
- Tribals have exclusive crafting recipes locked to them, like the list above these items make for more use when clearing a bunker as you can make use of mob drops and not just bunker loot entirely.
- Tribals are given bonuses for using primitive certain medical supplies like poultice and powder.

### Cons
- Tribals are restricted from using advanced machines due to lacking the understanding or skills required to use these things. Advanced machines like,

1.                - Ammo benches
2.                - Workshops
3.                - Sleepers (Editor note: already restricted via chem book)
4.                - Bio Gen
5.                - Hydro Trays
6.                - Etc...

This encourages having other roles to pair with tribals so that they can create the best of both worlds. Or having tribals act diplomatic and trade with others to gather supplies they cannot normally make themselves.  For example, A tribal and wastelander duo can make the most out of bunker loot given a waster can produce ammo, tools, and parts for weaponry, and a tribal can produce armors, and tools made from dead mobs to be resold or used for later purposes. 

~~### Mapping changes (**Slaughter Pass** region)~~
~~These changes only affect the mapping of the Tribe region and the tribe dungeon located inside of it. The tribe base currently has a good layout for worldbuilding and map exploration. I've decided to use the old tribe base as a small canyon pass that gives players a narrow passage in getting through zones and areas. Similar to the old shortcut players used to reach the NCR from Oasis, this serves that purpose as well as being a strategic area for any factions that wish to take over the pass.~~

MAPPING CHANGE OVERRULED, MAP CHANGES FOR REGION HERE.
https://github.com/LoneStarF13/LoneStar/pull/70

~~The pass will be a testament toward the Legion's ruthlessness, almost similar to how Tipton was a sign to strike fear into the hearts of those that witnessed it, so too will Slaughter pass take this role. Inside the pass are a few feral Legion dogs left behind, once the pass is cleared of the beasts, it is safe to traverse or settle along with it.~~
## Why It's Good For The Game

Currently, after watching an 80 pop round, not a single person played tribe. That right there is an issue of a faction that needs more player freedom and less reliance on set goals. This allows people who wish to play a more tribal RP-oriented playstyle to act in their own free ability, making their own tribe and their own stories. More player interaction is good, and opening up a closed section of the map for traffic is also a good move. More power to the individual, not the group.

## Pre-Merge Checklist

- [X] You tested this on a local server.
- [X] This code did not runtime during testing.
- [X] You documented all of your changes.

## Changelog

:cl:
add: Tribal role with their own mechanics and crafting exclusives.
add: New crafting recipes for tribals
add: Create tribe tab
del: Removed Tribal faction
tweak: loot in the tribe dungeon to a unique item
balance: move some tribal recipes to be locked behind the role
balance: technophobe has more blacklisted items
/:cl:

### CODE REVIEW

**Comment by OneOneThreeEight** [comment]
interesting, will be tracking how this goes.

**Comment by Tokorizo** [comment]
>4 errors

image

**Comment by Tokorizo** [comment]
Right, this should be ready for testing now.

File: code/modules/jobs/job_types/wasteland.dm [hunk 1]
```diff
@@ -1343,3 +1343,133 @@ datum/job/wasteland/f13dendoctor
...
+	H.mind.teach_crafting_recipe(/datum/crafting_recipe/tribalwar/sturdybow)
+	H.mind.teach_crafting_recipe(/datum/crafting_recipe/tribalwar/warclub)
+	H.mind.teach_crafting_recipe(/datum/crafting_recipe/tribalwar/silverbow)
+	H.mind.teach_crafting_recipe(/datum/crafting_recipe/tribalwar/arrowbone)
+	H.mind.teach_crafting_recipe(/datum/crafting_recipe/tribalwar/bonespear)
+	H.mind.teach_crafting_recipe(/datum/crafting_recipe/tribalwar/bonecodpiece)
+	H.mind.teach_crafting_recipe(/datum/crafting_recipe/tribalwar/bracers)
+	H.mind.teach_crafting_recipe(/datum/crafting_recipe/tribal/bonetalisman)
+	H.mind.teach_crafting_recipe(/datum/crafting_recipe/tribal/bonebag)
+	H.mind.teach_crafting_recipe(/datum/crafting_recipe/tribalwar/spearquiver)
```
Comment (Kapu1178) [inline code/modules/jobs/job_types/wasteland.dm:1475]: Store all of these in a list and just do

for(var/datum/crafting_recipe/recipe as() in list)
   H.mind.teach_crafting_recipe(recipe)

Comment (OneOneThreeEight) [reply code/modules/jobs/job_types/wasteland.dm:1475]: Agreed, makes the code a bit cleaner.

**Comment by cornercube** [comment]
i still think that the tribals should start out in a common zone instead of interspersed like wastelanders, to encourage tribals to stick together

**Comment by Neveralone1744** [comment]
id like to see a common area too. we even already picked a small spot that could work for us. a tribe scattered is a tribe dead

**Comment by Doru-7** [comment]
In response to the fact that an 80 pop round happened with no one playing tribal, there's a few key reasons for that. 
The first is that there's so much lore and mechanical specialization that's entirely original and not based on anything existing in fallout. This makes it hard for someone interested in playing as a tribal to play without having a wiki page open at all times.
The second is that all the players who played tribal characters and understood what they were have either wrapped up their stories with those characters and left the region, or straight up dropped those characters under the premise that the Legion have wiped them out. This is because they were kindly given several weeks of notice that the tribe would soon no longer exist, so they made their peace and left.
The third, and most troubling, is that the people who remain in the tribe(with the exception of a few veteran players who are still telling their stories to the very end.) are new players, with their own unique reasons for playing as a tribal. They no longer have guidance from veteran tribe players, meaning they can't learn the unique laws, traditions, and rituals in a natural manner. A few of these new players have even taken advantage of their faction's neutral stance to engage in TDM with the more populated factions, creating a small but significant sense of hostility towards the tribe as a whole.

This is all to say that Tribals are set on a precariously thin rope, and we who care about the tribes have to do our best to make sure it functions in a way that will ensure its survival. Therefore, I have two suggestions to add to this PR:
1. A common spawn location, so tribal players can group up easier. My recommendation is the tent and campfire at the end of the road that leads to the vault museum tileset, directly south of Oasis. The only issue is that it's a semi-common place for players to create settlements that may trap any new players that spawn in.
2. Book Binders and digital library. These are currently being used in the Brotherhood's base, but would benefit tribals who can find them if they appear in more places in the general wasteland. This is because the Wayfarers relied on oral tradition to teach new members. If a tribe of the future is to survive, they'll need written tradition as well. This means that all the lore associated with the tribes, be that the gods and traditions of the old Wayfarers, or the scribblings of the new tribes to come, will be easily accessible in a natural manner, and will even be available for outsiders to learn from.(Which also gives the Brotherhood scribes a reason to read books in the library, since new and functional books will be coming in.)

TL;DR: Tribals want to stick together. Common spawn area and the ability to write books will be a huge help.

**Review by OneOneThreeEight — CHANGES_REQUESTED** [review]
Address merge conflicts, please.

File: code/modules/jobs/job_types/tribals.dm [hunk 2]
```diff
@@ -6,7 +6,8 @@
 	enforces = "The tribe operate as if they are a part of a singular clan. Any harm dealt upon another member is likened to harm placed upon your brother or sister. Above all things, the tribe comes first."
 	objectivesList = list("Leadership recommends the following goal for this week: Recruit worthy outlanders into the tribe","Leadership recommends the following goal for this week: Hunt dangerous creatures to improve the clan's standing", "Leadership recommends the following goal for this week: Preserve dangerous technology to prevent it from falling into the wrong hands.")
 
-/datum/outfit/job/tribal/
+//Commented out for now
```
Comment (OneOneThreeEight) [inline code/modules/jobs/job_types/tribals.dm:9]: un-comment these out, please, they are messing with the ghost role spawners. jobs with 0 slots will never show up to the playerbase anyway

**Review by OneOneThreeEight — CHANGES_REQUESTED** [review]
